starfish/test/full_pipelines/cli/_base_cli_test.py

```diff
@@ -0,0 +1,34 @@
+import os
+import shutil
+import unittest
+from typing import List, Tuple
+
+from starfish.intensity_table import IntensityTable
+from starfish.util import exec
+
+
+class CLITest(unittest.TestCase):
```

Comment:
can this just be decorated with `unittest.skip`?
https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures

Reply:
It looks like thats for individual test within a test class? If I put the decorator on the test_run_pipieline method in the base class the subclasses also get skipped.

Reply:
https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures

`Classes can be skipped just like methods:`

Reply:
But the subclasses also get skipped if I add that decorator to the whole class

Reply:
Ah right, so do this instead:

1. make CLITest _not_ an instance of unittest.TestCase.
2. make each of the concrete test classes in `starfish/test/full_pipelines/cli` extend (CLITest, unittest.TestCase).